src/firecracker/src/main.rs

```diff
@@ -397,6 +398,7 @@ fn main_exec() -> Result<(), MainError> {
         });
 
         let start_time_cpu_us = arguments.single_value("start-time-cpu-us").map(|s| {
+            log_dev_preview_warning("start-time-cpu-us", "start");
```

Comment:
mh, not sure if "dev preview" is the correct message to print here :thinking: In Firecracker we just print deprecations as `warn!` logs, e.g. 
```suggestion
            warn("The --start-time-cpu-us argument is deprecated");
```

Reply:
doh! ok coffee is not making effect